docs/troubleshooting.md

```diff
@@ -73,3 +75,14 @@ entry section from `dhcpd_leases` file. Finally, reboot your system.
 
 **Note:** You can completely reset IP database by removing both the files
 manually but this is very **risky**.
+
+<a name="virtualbox-driver"></a>
+## virtualbox driver
+
+<a name="error-getting-state-for-host-machine-does-not-exist"></a>
+### Error: getting state for host: machine does not exist
+
+The problem is likely to be either you have a outdated virtualbox installed or
```

Comment:
If you are using Windows, the problem is likely to be either  an outdated version of Virtual Box or you forgot to use `--vm-driver virtualbox` option when starting _minishift_.